### PR TITLE
[IND-556] wait until bazooka image finishes upgrading before running bazooka in auxo lambda

### PR DESCRIPTION
**Pull request by dydxwill**
### Changelist
Wait until bazooka image finishes upgrading before running bazooka in auxo lambda.
When running auxo in our testnet environment, the old bazooka image was used, which led to an expected db migration not happening.

### Test Plan
ran in dev

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

File: indexer/services/auxo/src/index.ts [hunk 1]
```diff
@@ -117,17 +119,37 @@ async function upgradeBazooka(
...
       ImageUri: imageUri,
     }),
   );
-  logger.info({
-    at: 'index#upgradeBazooka',
-    message: 'Successfully upgraded bazooka',
-    response,
-  });
+
+  // Polling to check the update status
```
Comment (vincentwschau) [inline indexer/services/auxo/src/index.ts:129]:
```suggestion
  // Polling to check the update status, only continue after the bazooka lambda function has been succesfully updated
```

File: indexer/services/auxo/src/index.ts [hunk 2]
```diff
@@ -117,17 +119,37 @@ async function upgradeBazooka(
...
+        message: 'Successfully upgraded bazooka',
+        response: statusResponse,
+      });
+      return;
+    } else if (updateStatus === LastUpdateStatus.Failed) {
+      throw new Error('Failed to upgrade bazooka');
+    }
+
+    // Wait for 5s before checking again
+    await new Promise((resolve) => setTimeout(resolve, 5000));
```
Comment (vincentwschau) [inline indexer/services/auxo/src/index.ts:151]: nit: 1 minute timeout + exit the function?
Non-blocking

File: indexer/scripts/deploy-commit-to-env.sh [hunk 3]
```diff
@@ -32,6 +32,7 @@ printf "account: %s\n" $account
 
 dockerfile=Dockerfile.service.remote
 case $service in
+    "auxo") dockerfile=Dockerfile.auxo.remote;;
```
Comment (vincentwschau) [inline indexer/scripts/deploy-commit-to-env.sh:35]: isn't this already on line 37?

File: indexer/services/auxo/src/index.ts (hunk 1 above)
Comment (dydxwill) [reply indexer/services/auxo/src/index.ts:129]: done

File: indexer/services/auxo/src/index.ts (hunk 2 above)
Comment (dydxwill) [reply indexer/services/auxo/src/index.ts:151]: I added a 2 min timeout. Updating lambda image can be slow in my experience.

File: indexer/scripts/deploy-commit-to-env.sh (hunk 3 above)
Comment (dydxwill) [reply indexer/scripts/deploy-commit-to-env.sh:35]: removed

**Pull request merged**